turf/bq/test/turf_integration.js

```diff
@@ -0,0 +1,118 @@
+const assert = require('assert').strict;
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const BQ_PROJECTID = process.env.BQ_PROJECTID;
+const BQ_DATASET_TURF = process.env.BQ_DATASET_TURF;
+
+describe('TURF integration tests', () => {
+
+    let client;
+    before(async () => {
+        if (!BQ_PROJECTID) {
+            throw "Missing BQ_PROJECTID env variable";
+        }
+        if (!BQ_DATASET_TURF) {
+            throw "Missing BQ_DATASET_TURF env variable";
+        }
+        client = new BigQuery({projectId: `${BQ_PROJECTID}`});
+    });
+
+    it ('Returns the proper version', async () => {
+        const query = `SELECT \`${BQ_PROJECTID}\`.\`${BQ_DATASET_TURF}\`.VERSION() as versioncol;`;
+        let rows;
+        await assert.doesNotReject( async () => {
+            const [job] = await client.createQueryJob({ query: query });
+            [rows] = await job.getQueryResults();
+        });
+        assert.equal(rows.length, 1);
+        assert.equal(rows[0].versioncol, 1);
+    });
+
+    it ('BBOX should work', async () => {
```

Comment:
We should be exposing and testing things with geographies and forget/remove the GeoJSON API. Also, in general (not only this specific function) this is missing checking possible errors (NULL, invalid parameters, etc).